src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -0,0 +1,233 @@
+
+public class MealServletTest{
+    private final LocalServiceTestHelper helper =
+        new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+    private static final Meal MEAL_EMPTY = new Meal(
+        0L, "", "", new ArrayList<>(), "");
+    private static final Meal MEAL_1 = new Meal(
+        1L, "Fried potato", "Fried potato with mushrooms and onion.",
+            new ArrayList<>(Arrays.asList("potato", "onion", "oil")), "Main");
+    private static final Meal MEAL_1_DUPLICATE = new Meal(
+        1L, "Vegetable soup", "Vegetable soup with onion.",
+            new ArrayList<>(Arrays.asList("potato", "onion")), "Soup");
+    private static final Meal MEAL_2 = new Meal(
+        2L, "Chocolate cake", "Chocolate cake with butter cream and strawberry.",
+            new ArrayList<>(Arrays.asList("flour", "water", "butter", "strawberry")), "Dessert");
+    
+
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+    }
+
+    @After
+    public void tearDown() {
+        helper.tearDown();
+    }
+
+    // TODO(sandatsian): add more specific tests for search feature and get list feature.
+    // TODO(grenlayk): add test for get similar feature.
+    // Get one of the existing objects Meal by id.
+    @Test
+    public void getMealByIdTest() throws IOException, ServletException {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        ds.put(createMealEntity(MEAL_1));
+        ds.put(createMealEntity(MEAL_2));
+        
+        MealServlet servlet = new MealServlet();
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        request.setPathInfo("/2");
+        
+        try {
+            servlet.doGet(request, response);
+        } catch(Exception e) {
+            System.out.println(e);
+        }
+        Gson gson = new Gson();
+        String expected = gson.toJson(MEAL_2);
+        String actual = response.getContentAsString().trim();
+
+        assertEquals(expected, actual);
+    }
+
+    // Get an object from empty datastore.
+    // Result: Response Status NOT FOUND.
+    @Test
+    public void getMealByIdFromEmptyDsTest() throws IOException, ServletException {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        
+        MealServlet servlet = new MealServlet();
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        request.setPathInfo("/2");
+        
+        try {
+            servlet.doGet(request, response);
+        } catch(Exception e) {
+            System.out.println(e);
+        }
+        MockHttpServletResponse expectedResponse = new MockHttpServletResponse();
+        expectedResponse.setStatus(HttpServletResponse.SC_NOT_FOUND);
+        int expected = expectedResponse.getStatus();
```

Comment:
`HttpServletResponse.SC_NOT_FOUND` is just an `int`. So you can discard the `MockHttpServletResponse expectedResponse` entirely, and just do the following:
```
int actual = HttpServletResponse.SC_NOT_FOUND;
```
Or you can even remove this variable too, and just do:
```
assertEquals(HttpServletResponse.SC_NOT_FOUND, response.getStatus());
```

(Please apply this to all tests that expect a status response).